example_1_inventory/main.py

```diff
@@ -0,0 +1,52 @@
+from speakleash import Speakleash
```

Comment:
It would be good to add a script description at the top of the file (module docstring).
Minor cosmetic changes. Standard library imports should be placed at the top, and related third-party imports (from pip) should come after them, separated by empty lines:
"""Speakleash datasets inventory"""
import os

from speakleash import Speakleash
from termcolor import colored

Reply:
Fixed